Maple2.Server.Game/Model/Field/Actor/Actor.cs

```diff
@@ -73,13 +74,17 @@ public abstract class Actor<T> : IActor<T>, IDisposable {
         long damageAmount = 0;
         for (int i = 0; i < attack.Damage.Count; i++) {
             Reflect(caster);
-            targetRecord.AddDamage(DamageType.Normal, -2000);
-            damageAmount -= 2000;
+            targetRecord.AddDamage(DamageType.Normal, -50000);
+            damageAmount -= 50000;
         }
 
         if (damageAmount != 0) {
             long positiveDamage = damageAmount * -1;
-            DamageDealers.AddOrUpdate(caster.ObjectId, positiveDamage, (_, current) => current + positiveDamage);
+            if (!DamageDealers.TryGetValue(caster.ObjectId, out DamageRecordTarget? record)) {
```

Comment:
this was wrong?

Reply:
I think ultimately we should have this record with more info. For dungeons we'll need total hits, healed, etc which I think this can grow to eventually.
![image](https://github.com/AngeloTadeucci/Maple2/assets/62830952/c3180001-ec23-4823-9417-031d9b1cacdf)